avro-fastserde/regenerate_avro.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+mkdir -p avro_tools
+
+# Avro-1.4.1 could not handle `alias` properly, so we choose to use avro-tools-1.4.0 to generate specific record
+# to ignore `alias` fields.
+DEFAULT_AVRO_TOOLS_140_JAR="avro_tools/avro_tools_140/avro-tools-1.4.0.jar"
+DEFAULT_AVRO_TOOLS_177_JAR="avro_tools/avro_tools_177/avro-tools-1.7.7.jar"
+DEFAULT_AVRO_TOOLS_182_JAR="avro_tools/avro_tools_182/avro-tools-1.8.2.jar"
+
+if [ ! -f $DEFAULT_AVRO_TOOLS_140_JAR ]; then
+  wget "https://repo1.maven.org/maven2/org/apache/avro/avro/1.4.0/avro-tools-1.4.0.jar" -P avro_tools/avro_tools_140
+fi
+if [ ! -f $DEFAULT_AVRO_TOOLS_177_JAR ]; then
+  wget "https://repo1.maven.org/maven2/org/apache/avro/avro-tools/1.7.7/avro-tools-1.7.7.jar" -P avro_tools/avro_tools_177
+fi
+if [ ! -f $DEFAULT_AVRO_TOOLS_182_JAR ]; then
+  wget "https://repo1.maven.org/maven2/org/apache/avro/avro-tools/1.8.2/avro-tools-1.8.2.jar" -P avro_tools/avro_tools_182
+fi
+
+AVRO_SCHEMAS_PATH=(
+  "src/test/resources/avro/fastserdetest.avsc"
+  "src/test/resources/avro/defaultsTest.avsc"
+  "src/test/resources/avro/stringableTest.avsc"
+)
+CODE_GEN_PATH=(
```

Comment:
could we add a comment explaining what the intended purpose of this structure is? at 1st glance it looks like a copy-paste error

Reply:
Left some comments to explain the purpose, and let me know if they are not good enough.